### PR TITLE
Adjust formatting and namespace scope for SpatialConstants

### PR DESCRIPTION
**Pull request by improbable-valentyn**
**Contributions**: We are not currently taking public contributions - see our [contributions](CONTRIBUTING.md) policy. However, we are accepting issues and we do want your [feedback](../README.md#give-us-feedback).

-------

#### Description
Adjusted format and namespace scope to be consistent.

#### Primary reviewers
@m-samiec

### CODE REVIEW

File: SpatialGDK/Source/SpatialGDK/Public/SpatialConstants.h [hunk 1]
```diff
@@ -27,42 +42,6 @@ enum ESchemaComponentType : int32
...
-	Invalid,
-	ClientReliable,
-	ClientUnreliable,
-	ServerReliable,
-	ServerUnreliable,
-	NetMulticast,
-	CrossServer
-};
-
-FORCEINLINE ERPCType FunctionFlagsToRPCType(EFunctionFlags FunctionFlags)
```
Comment (aleximprobable) [inline SpatialGDK/Source/SpatialGDK/Public/SpatialConstants.h]: Is this function not needed anymore?

File: SpatialGDK/Source/SpatialGDK/Public/SpatialConstants.h [hunk 2]
```diff
@@ -12,6 +12,21 @@
 
 #include "SpatialConstants.generated.h"
 
+UENUM()
```
Comment (aleximprobable) [inline SpatialGDK/Source/SpatialGDK/Public/SpatialConstants.h]: Why is `ERPCType` not in `SpatialConstants` namespace anymore?

Comment (improbable-valentyn) [reply SpatialGDK/Source/SpatialGDK/Public/SpatialConstants.h]: It's a UENUM, so it has to be in global namespace.

Comment (improbable-valentyn) [reply SpatialGDK/Source/SpatialGDK/Public/SpatialConstants.h]: And it has to be a UENUM to be available from the editor UI.

File: SpatialGDK/Source/SpatialGDK/Public/SpatialConstants.h (hunk 1 above)
Comment (improbable-valentyn) [reply SpatialGDK/Source/SpatialGDK/Public/SpatialConstants.h]: Correct.

**Pull request merged**